sdk/core/System.ClientModel/gen/NetStandardPolyfil/IsExternalInit.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ComponentModel;
+
+namespace System.Runtime.CompilerServices
+{
+    /// <summary>
+    /// Reserved to be used by the compiler for tracking metadata.
+    /// This class should not be used by developers in source code.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+#if SYSTEM_PRIVATE_CORELIB
+    public
+#else
+    internal
+#endif
+    static class IsExternalInit
```

Comment:
Make sure you typeforward this otherwise your callers will run into issues when compiling against a framework without it, then running on one with it.  The internal location of the type leaks to callers in your public API.

Reply:
Maybe you don't care since this is only used in the source generator?  Still - be careful when polyfilling this type, best to include an ifdef and typeforward for later frameworks.